openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -231,8 +231,8 @@ def test_create_account(self):
             'bio': None,
             'profile_image': {
                 'has_image': False,
-                'image_url_full': 'http://example-storage.com/profile_images/default_50.jpg',
-                'image_url_small': 'http://example-storage.com/profile_images/default_10.jpg',
+                'image_url_full': '/static/default_50.png',
+                'image_url_small': '/static/default_10.png',
```

Comment:
I've tested this on devstack and on my sandbox, and full URLs are returned. I'm not entirely sure why the loader doesn't return the host part in testing, but I don't think it is something to be concerned with.